01week/rockPaperScissors.js

```diff
@@ -8,10 +8,26 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(a, b) {
 
-  // Write code here
+  var hand1 = a.toLowerCase().trim();
+  var hand2 = b.toLowerCase().trim();
 
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  } else if (hand1 === "rock" && hand2 === "paper") {
+    return "Hand two wins!";
+  } else if (hand1 === "rock" && hand2 === "scissors") {
+    return "Hand one wins!";
+  } else if (hand1 === "paper" && hand2 === "rock") {
+    return "Hand one wins!";
+  } else if (hand1 === "paper" && hand2 === "scissors") {
+    return "Hand two wins!";
+  } else if (hand1 === "scissors" && hand2 === "paper") {
+    return "Hand one wins!";
+  } else if (hand1 === "scissors" && hand2 === "rock") {
+    return "Hand two wins!";
+  }
 }
```

Comment:
This is great, but just to give you another way to think of this, could you write only two else if statements using ||?